src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -16,12 +17,9 @@ export default function WebexLocalMedia({meetingID}) {
   const {localVideo} = useMeeting(meetingID);
   const {ID} = useMe();
   const videoRef = useStream(localVideo);
+  const disabledVideo = ID ? <WebexAvatar personID={ID} /> : <Spinner />;
```

Comment:
Can you add a snapshot test for the loading state?

Reply:
not sure how to mock `useMe()` to return undefined.